projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -307,6 +307,7 @@ export interface FieldType {
     pipeArgs?: IFieldPipeArgs;
     defaultTimeFormat?: string;
     defaultDateTimeFormat?: string;
+    validators?: any[];
```

Comment:
This is still the base for `ColumnType` right? So this change would affect the the grid and we might have to consider that in the way we expose it for the QB.
Also not sure `any[]` is a good type for this, technically it's `IFieldValidator[]` from what I see. Guessing that's because the column already has an _internal_ `validators: Validator[]` which doesn't match and would error out (that should be a hint).
While the declarative `IFieldValidator` would actually be better for Xplat purposes,  it could be a bit limiting to the validators we support without a mechanism to extend them. This might force us to consider the options for the column validation on xplat as well to keep those in sync.